avocado/plugins/yaml_to_mux.py

```diff
@@ -262,8 +268,12 @@ def run(self, args):
             try:
                 args.mux.data_merge(create_from_yaml(multiplex_files, debug))
             except IOError as details:
-                logging.getLogger("avocado.app").error(details.strerror)
-                sys.exit(exit_codes.AVOCADO_JOB_FAIL)
+                if getattr(args, "mux_ignore_missing", 'off') == 'on':
+                    msg = 'Cannot access the multiplex file(s) in this host.'
```

Comment:
Do you think this message is necessary? I think when one specifies `--mux-ignore-missing` they know what are they doing... Anyway I don't say no to this, it just does not seem necessary for me.